test/bin/manage_brew_rpms.sh

```diff
@@ -38,10 +39,17 @@ action_download() {
 
     # Attempt downloading the specified build version
     local package
-    package=$(brew list-builds --quiet --package=microshift --state=COMPLETE | grep "^microshift-${ver}" | sort | tail -1) || true
+    if [ "${ver_type}" = "zstream" ]; then
```

Comment:
Nit: This might be more readable if we use `case "${ver_type}"`

Reply:
I agree, added it in https://github.com/openshift/microshift/pull/4900/commits/6c39fc71cba995ec14b8c546708adc2de63e9b6a commit